command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
```

Comment:
On Thu, Jul 28, 2016 at 11:42:26AM -0700, Doug Davis wrote:
> > @@ -0,0 +1,189 @@
> > +# Operations
> > +
> > +A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
> 
> why the empty `[]` before the` MUST`?

Markdown reference link [1]:

  The implicit link name shortcut allows you to omit the name of the
  link, in which case the link text itself is used as the name. Just
  use an empty set of square brackets…

[1]: http://daringfireball.net/projects/markdown/syntax#link